src/cmd/cmd_query.go

```diff
@@ -270,13 +291,13 @@ func RunQueryCmdLine() {
 		sybil.Debug("USING QUERY SPEC", querySpec)
 
 		start := time.Now()
-		// We can load and query at the same time
```

Comment:
you don't like my comments? :P

Reply:
heh, cherry pick conflict resolution artifact